viewing_party/party.py

```diff
@@ -1,23 +1,194 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
+def create_movie(movie_title, genre, rating):
+
+    movie= {}
+
+    if movie_title and genre and rating:
+    
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating          
+        return movie
+
+    return None

```

Comment:
One way to shorten this could be like the following: 
```python
def create_movie(movie_title, genre, rating):

    if not movie_title or not genre or not rating:
        return None
	
	return {
		"title": movie_title,
		"genre": genre,
		"rating": rating
	}
```
I would be okay with this since all we are doing is creating a dictionary representation of a movie and then return it.